aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -135,24 +137,48 @@ When swapping between deployment slots, any system using data protection won't b
 
 For more information, see <xref:security/data-protection/implementation/key-storage-providers>.
 <a name="deploy-aspnet-core-preview-release-to-azure-app-service"></a>
-<!-- revert this after 3.0 supported
-## Deploy ASP.NET Core preview release to Azure App Service
```

Comment:
Just making sure that this is your plan: This section was always kept here to cover the **_next preview release_**. Next, it would go to cover preview of 3.1. If you're removing it tho permanently ... cool ... just calling out why it was always here, even when a release occurs.